.github/workflows/android.yml

```diff
@@ -230,9 +233,140 @@ jobs:
           path: ${{ env.SCCACHE_ERROR_LOG }}
           if-no-files-found: 'ignore'
 
+      - name: Prepare build and test artifacts (split)
+        run: |
+          # Prepare directories
+          mkdir -p ${INSTALL_TEST_DIR}/bin/intel64/Release
+          mkdir -p ${INSTALL_DIR}/lib
+
+          # Install tests into tests dir
+          cmake --install ${BUILD_DIR} --config ${{ env.BUILD_TYPE }} --prefix ${INSTALL_TEST_DIR} --component tests
+
+          # Install runtime into install dir; fallback to full install if component is unavailable
+          cmake --install ${BUILD_DIR} --config ${{ env.BUILD_TYPE }} --prefix ${INSTALL_DIR} --component core \
+            || cmake --install ${BUILD_DIR} --config ${{ env.BUILD_TYPE }} --prefix ${INSTALL_DIR}
+
+          # If tests were installed under tests/, move CPU test binaries to expected path
+          if [ -d "${INSTALL_TEST_DIR}/tests" ]; then
+            echo "Moving tests from ${INSTALL_TEST_DIR}/tests/ to ${INSTALL_TEST_DIR}/bin/intel64/Release/"
+            find ${INSTALL_TEST_DIR}/tests -name "*cpu*test*" -type f -exec cp {} ${INSTALL_TEST_DIR}/bin/intel64/Release/ \;
+          fi
+
+          # Add TBB and libc++ shared to runtime package
```

Comment:
Looks a bit hacky, why this is needed?

Reply:
- TBB is built separately and is not shipped as a part of OV package
- libc++_shared.so is not available inside the emulator, but required to run the app and it is a part of the toolchain

I haven't found a less hacky way to ship them there